api/rest/triple.yaml

```diff
@@ -0,0 +1,109 @@
+                    example: '{"dc/bsmvthtq89217":[{"subjectId":"dc/bsmvthtq89217","predicate":"country","objectId":"country/AUS","objectName":"Australia","objectTypes":["Country"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"continent","objectId":"oceania","objectName":"Oceania","objectTypes":["Continent"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"typeOf","objectId":"BiologicalSpecimen","objectName":"BiologicalSpecimen","objectTypes":["Class"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"provenance","objectId":"dc/93qydx3","objectName":"http://nybg.org/","objectTypes":["Provenance"],"provenanceId":"dc/93qydx3"}]}'
+        '400':
+          description: Missing argument 'dcids'.    
+    post:
+      summary: Given a list of nodes, return triples which are associated with the
+        specified node(s).
+      description: Intended for users who would like to modify only the bodies of
+        their cURL commands.
+      security:
+        - api_key: []
+      requestBody:
+        required:
+          - dcids
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/PostBody'
+            example:
+              dcids: ["dc/bsmvthtq89217"]
+              limit: 1
+      produces:
+      - application/json
+      responses:
+        '200':
+          description: A string encoding of a JSON object keyed by the given DCIDs with the value a list of associated triples.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  payload:
+                    type: string
+                    example: '{"dc/bsmvthtq89217":[{"subjectId":"dc/bsmvthtq89217","predicate":"country","objectId":"country/AUS","objectName":"Australia","objectTypes":["Country"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"continent","objectId":"oceania","objectName":"Oceania","objectTypes":["Continent"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"typeOf","objectId":"BiologicalSpecimen","objectName":"BiologicalSpecimen","objectTypes":["Class"],"provenanceId":"dc/93qydx3"},{"subjectId":"dc/bsmvthtq89217","predicate":"provenance","objectId":"dc/93qydx3","objectName":"http://nybg.org/","objectTypes":["Provenance"],"provenanceId":"dc/93qydx3"}]}'
+        '400':
+          description: Missing argument 'dcids'.
+securityDefinitions:
+  api_key:
+    type: apiKey
+    name: api_key
+    description: Optional.
+definitions:
+  PostBody:
+    type: object
+    required:
+    - dcids
+    properties:
+      dcids:
+        type: array
+        items:
+          type: string
+        description: DCIDs targeted by the query.
+      limit:
+        type: integer
+        description: Limit on number of triples desired in response.
+      key:
+        type: string
+        description: API key.
```

Comment:
this is an optional param - can we mark it as such, or add it to the description: "API key (optional)."